packages/connectors/package.json

```diff
@@ -17,7 +17,9 @@
     "ts:check": "tsc --noEmit"
   },
   "peerDependencies": {
-    "fuels": ">=0.77.0"
+    "fuels": ">=0.77.0",
+    "socket.io-client": "4.7.2",
+    "axios": "1.5.1"
```

Comment:
For this deps, they should be under `dependencies` and not peer as this will require user to also install this with `@fuels/connectors` what is not desired.

Reply:
And in the best case it should be under the package only not on this umbrella. You can move all there and if we see the problem is on our current infra we will help by improving the infra.

Reply:
?

Reply:
we talked offline about this approach last week, however, let's keep only `fuels` here for now